version/VERSION

```diff
@@ -1 +1 @@
-1.15.4
\ No newline at end of file
+1.15.3
```

Comment:
shouldn't it be `1.15.3-criteo` here?

Reply:
we have to do that when publishing a version (within the changelog PR).
We can call it `1.15.3-criteo-dev` if you **really want to** but we don't have that much development on this repo.

Reply:
After the issue we had yesterday, maybe `1.15.3-criteodev` (or whatever, just not criteo1) is the way to go for this PR